pkg/platform/kube/platform.go

```diff
@@ -558,13 +558,21 @@ func (p *Platform) RedeployFunction(ctx context.Context, redeployFunctionOptions
 		&permissionOptions); err != nil {
 		return errors.Wrap(err, "Failed authorizing OPA permissions for resource")
 	}
+	var state functionconfig.FunctionState
+
+	switch redeployFunctionOptions.DesiredState {
```

Comment:
Add a comment that setting a different function state is not supported so we fallback to ready.